src/test/java/org/jboss/xavier/integrations/EndToEndTest.java

```diff
@@ -182,6 +185,18 @@
     @Value("${test.bigfile.vms_expected:5254}")
     private int numberVMsExpected_InBigFile;
 
+    @Inject
+    CamelContext camelContext;
+
+    @Inject
+    AmazonS3 amazonS3;
```

Comment:
I think this import is not used anywhere

Reply:
+1